README.md

```diff
@@ -11,6 +11,21 @@
 
 A standalone (detached slate) block that provides a text block meant to be used as introduction to a content page. This text usually having special formatting. It allows different paragraphs using breaklines (ENTER). It does not allow certain styling (as headings).
 
+
+## Block configuration
+
+### `unwantedButtons`
+
+It allows you to specify which buttons from the current slate block toolbar should be removed from the introduction block.
```

Comment:
@sneridagh We should also mention the default here.

Reply:
Done.